src/components/landing-page/use-navigate-to-production.ts

```diff
@@ -1,17 +1,28 @@
 import { useEffect } from "react";
 import { useNavigate } from "react-router-dom";
 import { TJoinProductionOptions } from "../production-line/types.ts";
+// import { isMobile } from "../../bowser.ts";
 
 // Navigates to a production line as soon as a new production is pushed to the global state
 export const useNavigateToProduction = (
   joinProductionOptions: TJoinProductionOptions | null
 ) => {
   const navigate = useNavigate();
 
+  // useEffect(() => {
```

Comment:
same as above applies here

Reply:
[test: updated code to allow multiple calls on mobile](https://github.com/Eyevinn/intercom-frontend/pull/225/commits/d2d0f298d624c6cc50b110bdf96a3ef4ab157b08) is only to test if it worked on mobile